### DISCUSSION TITLE
Suggesstion for "Debug With Chrome"

### DISCUSSION THREAD

**Discussion by Bayramito** [software-mansion/react-native-reanimated#2455] [General]
Hello,

Im. using the latest stable release 2.2.2 right now but i strongly need that "Debug With Chrome" feature, do you have any suggesstions for me to do? Thank you

**Comment by terrysahaidak** [comment]
You can use flipper and debug your code directly. Also, it's possible to connect to hermes from chrome
https://reactnative.dev/docs/hermes#debugging-js-on-hermes-using-google-chromes-devtools

**Comment by Bayramito** [comment]
thank you but decided to use safari dev tools.
